try:
    a = 1
except Error1:
    pass
except:
    print("Unknown")
    raise
else:
    pass
finally:
    pass
pass
